proto/api.proto

```diff
@@ -542,6 +553,27 @@ message Ticker {
   Project project = 7;
 }
 
+message GetTickersRequestV2 {
```

Comment:
we don't need a separate endpoint for this. The new behavior should be added to the current `GetTickers` endpoint